kubernetes/samples/scripts/create-rcu-schema/drop-rcu-schema.sh

```diff
@@ -65,11 +63,13 @@ if [ -z ${namespace} ]; then
 fi
 
 if [ -z ${sysPassword} ]; then
-  sysPassword="Oradoc_db1"
+  echo "${script}: -q <SYSDBA user password> must be specified."
+  usage 1
 fi
 
 if [ -z ${schemaPassword} ]; then
```

Comment:
IIRC `[ -z ]` will not parse, so `${schemaPassword}` must be encased in quotes: `[ -z "${schemaPassword}"]`.

Reply:
I didn't change that line; if it didn't work, wouldn't it already have failed?

Reply:
this code could not be activated before since the password had a default (it was never empty)
without fixing this, it'll report a syntax error and continue if no password is specified

Reply:
@russgold Even though you didn't change this, can you please fix this by adding double quotes, as Tom has mentioned?